spec.md

```diff
@@ -180,6 +180,7 @@ The `Docker-Content-Digest` header, if present on the response, returns the cano
 If the digest does differ, it MAY be the case that the hashing algorithms used do not match.
 See [Content Digests](https://github.com/opencontainers/image-spec/blob/v1.0.1/descriptor.md#digests) <sup>[apdx-3](#appendix)</sup> for information on how to detect the hashing algorithm in use.
 Most clients MAY ignore the value, but if it is used, the client MUST verify the value against the uploaded blob data.
```

Comment:
I agree with the comment below that "uploaded blob data" isn't right for a manifest.

```suggestion
Most clients MAY ignore the value, but if it is used, the client MUST verify the value matches the returned manifest.
```